code/modules/mob/living/say.dm

```diff
@@ -519,11 +525,13 @@ var/list/headset_modes = list(
 	if(stuttering || (undergoing_hypothermia() == MODERATE_HYPOTHERMIA && prob(25)) )
 		speech.message = stutter(speech.message)
 
-	if(reagents && reagents.has_any_reagents(HYPERZINES))
-		speech.message = replacetext(speech.message," ","") // motor mouth
-		speech.message = replacetext(speech.message,",","") // motor mouth
-		speech.message = replacetext(speech.message,";","") // motor mouth
-		speech.message = replacetext(speech.message,"-","") // motor mouth
+	if (reagents)
+		var/datum/reagent/hyperzine/H = reagents.get_reagent_by_type(/datum/reagent/hyperzine)//also checks for hyperzine subtypes like cocaine etc
+		if (H && (H.data != "no motor mouth"))
+			speech.message = replacetext(speech.message," ","") // motor mouth
+			speech.message = replacetext(speech.message,",","") // motor mouth
+			speech.message = replacetext(speech.message,";","") // motor mouth
+			speech.message = replacetext(speech.message,"-","") // motor mouth
```

Comment:
All hyperzine users should suffer the plight of motor mouth, even if they're cultists.

Reply:
It's not hyperzine technically, but we don't have any other mechanics in the game that lets humans bypass slowdown to my knowledge so I'd rather piggyback that one than code a new one from scratch just for that one rune.